testsuite/features/step_definitions/common_steps.rb

```diff
@@ -5,11 +5,6 @@
 require 'securerandom'
 require 'pathname'
 
-When(/^I save a screenshot as "([^"]+)"$/) do |filename|
```

Comment:
This step is never referenced but is probably convenient for debugging purposes.